extensions/fairifier-rdf-extension/dtls/fairifier/PostFairDataToFairDataPoint.java

```diff
@@ -36,264 +41,171 @@
 
 public class PostFairDataToFairDataPoint extends Command {
 
-    private SimpleValueFactory f;
-    private static final Logger LOGGER = LoggerFactory.getLogger(PostFairDataToFairDataPoint.class);
+    private static SimpleValueFactory f;
 
     @Override
     public void doPost(HttpServletRequest req, HttpServletResponse res)
             throws ServletException, IOException {
         f = SimpleValueFactory.getInstance();
 
-        ArrayList<IRI> datasetThemes = new ArrayList<IRI>();
-        ArrayList<IRI> catalogThemes = new ArrayList<IRI>();
         String uuid_catalog = UUID.randomUUID().toString();
         String uuid_dataset = UUID.randomUUID().toString();
         String uuid_distribution = UUID.randomUUID().toString();
-        ArrayList<Literal> keyWords = new ArrayList<Literal>();
-        Agent agent = new Agent();
-        Date date = new Date();
-
-        String catalogString = null;
-        String datasetString = null;
-        String distributionString = null;
-
-        Resource r = null;
-
-        CatalogMetadata catalogMetadata = new CatalogMetadata();
-        DatasetMetadata datasetMetadata = new DatasetMetadata();
-        DistributionMetadata distributionMetadata = new DistributionMetadata();
 
         try {
             StringBuffer jb = new StringBuffer();
             String line = null;
             BufferedReader reader = req.getReader();
-            while ((line = reader.readLine()) != null)
+            while ((line = reader.readLine()) != null) {
                 jb.append(line);
-
+            }
             JSONObject fdp = new JSONObject(jb.toString()).getJSONObject("metadata");
             JSONObject catalog = fdp.getJSONObject("catalog");
             JSONObject dataset = fdp.getJSONObject("dataset");
             JSONObject distribution = fdp.getJSONObject("distribution");
             JSONObject uploadConfiguration = fdp.getJSONObject("uploadConfiguration");
 
+            CatalogMetadata catalogMetadata = null;
+            String catalogPost = null;
             if (!catalog.getBoolean("_exists")) {
+                String name = catalog.getString("http://purl.org/dc/terms/title").replace(" ", "_") + "_"
```

Comment:
`"http://purl.org/dc/terms/title"` appears/is-repeated 8 times in this class. Candidate for a constant.